app/views/course_user_data/show.html.erb

```diff
@@ -9,7 +17,11 @@
     Section: <strong><%= @requestedUser.section %></strong><br>
     <%= @requestedUser.school %> <%= @requestedUser.major %>, Class of <%= @requestedUser.year %></li>
   <% if @cud.instructor? %>
-  <li><b>Course Average Tweak</b> of <%= raw tweak(@requestedUser.tweak) %></li>
+    <li><b>Course Average Tweak</b> of <%= raw tweak(@requestedUser.tweak) %>
+    <% if @requestedUser.dropped? %>
```

Comment:
Is there a reason why there's no instructor check here too?

Reply:
it's within a section that can only be seen by an instructor (line 19)

Reply:
Oops mb